fpocket/protocols/protocol_mdpocket_findPockets.py

```diff
@@ -0,0 +1,616 @@
+            Plugin.runScript(self, 'extractISOPdb.py', args=self._getselIsovalueFreqArgs(), cwd=scriptDir)
+
+        self.cleanUp(scriptDir)
+
+    def pocketDefOutputStep(self):
+        pocketsDir = self._getExtraPath('pocketDetection')
+        if not self.keepDefaultFiles.get():
+            defaultFiles = [f'{pocketsDir}/mdpout_dens_iso_8.pdb', f'{pocketsDir}/mdpout_freq_iso_0_5.pdb']
+            for f in defaultFiles:
+                if os.path.exists(f):
+                    os.remove(f)
+
+    def mdPocketCharactStep(self):
+        if self.useSystem.get():
+            self.runMDPocket()
+        else:
+            self.runMDPocketPDB()
+
+        mdpocketDir = os.path.abspath(os.path.join(Plugin.getVar(FPOCKET_DIC['home']), 'bin'))
+        self.cleanUp2(mdpocketDir)
+
+    def defineOutputStep(self):
+        pocketsDir = self._getExtraPath('pocketCharacterization')
+        pocketFiles = os.listdir(pocketsDir)
+
+        self._cleanupDefaultFile(pocketsDir)
+        proteinFile = self._getProteinFile()
+
+        outputRois = SetOfStructROIs(filename=self._getPath('pockets.sqlite'))
+
+        for pFile in pocketFiles:
+            if 'mdpocket.pdb' not in pFile:
+                continue
+            self._processPocketFile(pFile, pocketsDir, proteinFile, outputRois)
+
+        outputRois.buildPDBhetatmFile()
+        self._defineOutputs(outputROIs=outputRois)
+
+
+
+    # --------------------------- INFO functions -----------------------------------
+    def _summary(self):
+        summary = ["If different isovalues were selected, the new pdb files appear in the 'extra/pocketDetection' folder."]
+        return summary
+
+    def _methods(self):
+        methods = []
+        return methods
+
+    def _warnings(self):
+        warnings = []
+        return warnings
+
+    # --------------------------- UTILS functions -----------------------------------
+    def getCoords(self):
+        pdbFile = os.path.abspath(self._getExtraPath('mdpoutput-{}.pdb'.format(str(self.isoValue.get()))))
+        coords = []
+        for pdbLine in open(pdbFile):
+            line = pdbLine.split()
+            coord = line[5:8]
+            coord = list(map(float,coord))
+            coords.append(coord)
+        return coords
+
+    def createPocketFile(self, clust, i):
+        outFile = self._getExtraPath('pocketFile_{}.pdb'.format(i+1))
+        with open(outFile, 'w') as f:
+            for j, coord in enumerate(clust):
+                f.write(writePDBLine(['HETATM', str(j + 1), 'APOL', 'STP', 'C', '1', *coord, 1.0, 0.0, '', 'Ve']))
+
+            f.write('\nEND')
+        return outFile
+
+    def moveFiles(self):
+        trajFile = self.inputSystem.get().getTrajectoryFile()
+        trajectory = os.path.abspath((trajFile))
+        if (self.inputSystem.get().getClass() == 'GromacsSystem'):
```

Comment:
At least for me, this is not working. self.inputSystem.get().getClass() would be something like <class 'gromacs.objects.data.GromacsSystem'> so != 'GromacsSystem.
try instead: str(type(self.inputSystem.get()).__name__)